index.html

```diff
@@ -0,0 +1,231 @@
+                        <a href="#">
+                            <p class="value">$200M</p>
+                        </a>
+                        <a href="#">
+                            <p class="text">SERIES E</p>
+                        </a>
+                    </div>
+                    <div class="center-box flex">
+                        <a href="#"><i class="fas fa-tag"></i>internet</a>
+                        <a href="#"><i class="fas fa-briefcase">Boyu Capital, +4</i></a>
+                    </div>
+                    <div class="bottom-box">
+                        <a href="#">
+                            <p>SoftBank-backed travel platform Klook raises $200M amid COVID-19</p>
+                            <span>on TechCrunch</span>
+                        </a>
+                    </div>
+                </div>
+
+                <div class="box" id="box-4">
+
+                    <div class="top-box">
+                        <img src="img/funding-img-4.jpg" alt="Wolt">
+                        <a href="#">
+                            <h4 class="orange">Wolt</h4>
+                        </a>
+                        <a href="#">
+                            <p class="value">$530M</p>
+                        </a>
+                        <a href="#">
+                            <p class="text">VENTURE CAPITAL</p>
+                        </a>
+                    </div>
+                    <div class="center-box flex">
+                        <a href="#"><i class="fas fa-tag"></i>Mobile, +5</a>
+                        <a href="#"><i class="fas fa-briefcase"> DST Global, +9</i></a>
+                    </div>
+                    <div class="bottom-box">
+                        <a href="#">
+                            <p>Food delivery startup Wolt raises $530M after tripping revenues</p>
+                            <span>on SiliconANGLE</span>
+                        </a>
+                    </div>
+                </div>
+            </div>
+        </section>
+    </main>
+    <footer>
+        <div class="footer-info ">
+            <div class="flex">
+                <a href="#"><i class="fab fa-facebook-square"></i></a>
+                <a href="#"><i class="fab fa-twitter"></i></a>
+                <a href="#"><i class="fab fa-youtube"></i></a>
+                <a href="#"><i class="fab fa-instagram-square"></i></a>
+                <a href="#"><i class="fab fa-pinterest-p"></i></a>
+                <a href="#"><i class="fa fa-rss"></i></a>
+                <a href="#"><i class="fas fa-envelope"></i></a>
+            </div>
+            <div class="footer-links">
+                <ul>
+                    <li><a href="#">EVENTS</a></li>
+                    <li><a href="#">ABOUT</a></li>
+                    <li><a href="#">ADVERTISE</a></li>
+                    <li><a href="#">JOBS</a></li>
+                    <li><a href="#">CONTACT</a></li>
+                </ul>
+            </div>
```

Comment:
- [ ] [Optional] The highlighted items should have a proper text size
### your page👇 
![image](https://user-images.githubusercontent.com/64360974/114019110-b1b62800-9876-11eb-860c-824e8628868f.png)

### original page👇 

![image](https://user-images.githubusercontent.com/64360974/114019321-ef1ab580-9876-11eb-8d8d-4f8136e1b00e.png)